packages/frontend/src/shared/api/axios.ts

```diff
@@ -2,6 +2,10 @@ import axios from 'axios'
 
 import { BASE_URL } from '@shared/config'
 
-axios.defaults.baseURL = BASE_URL ?? ''
+if (BASE_URL === undefined) {
+  throw new Error('Переменная BASE_URL незадана')
```

Comment:
```suggestion
  throw new Error('env BASE_URL is undefined')
```
Можно сделать нестрогую проверку на null, чтобы на оба проверял.